pkg/controller/controller_instance_test.go

```diff
@@ -898,6 +948,9 @@ func TestReconcileInstanceWithFailureCondition(t *testing.T) {
 	assertNumEvents(t, events, 0)
 }
 
+// TestPollServiceInstanceInProgressProvisioningWithOperation tests polling an instance
+// that is already in process of provisioning (background/asynchronously) and is still in
+// progress (should be re-polled)
 func TestPollServiceInstanceInProgressProvisioningWithOperation(t *testing.T) {
 	fakeKubeClient, fakeCatalogClient, fakeBrokerClient, testController, sharedInformers := newTestController(t, fakeosb.FakeClientConfiguration{
 		PollLastOperationReaction: &fakeosb.PollLastOperationReaction{
```

Comment:
This test has a catalog client action for which there is no assertion - please add.